### ISSUE TITLE
setTransactionName not implemented in scope

### ISSUE THREAD

**Issue by MattIPv4** [robertcepa/toucan-js#109]
`sentry.getScope().setTransactionName('whatever');` results `TypeError: sentry.getScope(...).setTransactionName is not a function`.

**Comment by MattIPv4** [comment]
I ended up monkey-patching this in at runtime for myself, so I could make use of it:

```js
const scopeProto = Object.getPrototypeOf(sentry.getScope());
scopeProto.setTransactionName = function (name) {
    this.adapter.setTransactionName(name);
};

const adapterProto = Object.getPrototypeOf(sentry.getScope().adapter);
const apply = adapterProto.applyToEventSync;
adapterProto.applyToEventSync = function (event) {
    const applied = apply.call(this, event);
    if (this._transactionName) applied.transaction = this._transactionName;
    return applied;
};
```

**Comment by robertcepa** [comment]
Thanks for filing! This should be fixed now in [3.0.0](https://github.com/robertcepa/toucan-js/releases/tag/toucan-js%403.0.0 )

Please reopen if this is still an issue.
